esmvaltool/utils/cmorizers/obs/cmor_config/Landschutzer2014.yml

```diff
@@ -0,0 +1,42 @@
+---
+# project at hand
+proj:
+  dataset: 'Landschutzer2014'
+  version: 'CDIAC_G05'
+  realm: 'clim'
+  field: 'TO2M'
+  frequency: {
+        'fgco2': 'Omon',
+        'spco2': 'Omon',
+        'dpco2': 'Omon',
+    }
+  metadata_attributes:
+    tier: '2'
+    source: 'CRESCENDO WP5 (CMCC)'
+    comment: 'cmorized for ESMValTool v2'
+    CMORconventions: 'CF/CMOR3'
+    CMORcreated: ''
+
+# Match CMOR variables with input file one(s)
+VAR_TO_CMOR:
+  fgco2: 'fgco2_smoothed' 
+  spco2: 'spco2_smoothed' 
+  dpco2: 'dco2_smoothed' 
+
+# specific raw file names (implicitly assumed to be .nc)
+VAR_TO_FILENAME:
+  fgco2: 'spco2_1998-2011_ETH_SOM-FFN_CDIAC_G05'
+  spco2: 'spco2_1998-2011_ETH_SOM-FFN_CDIAC_G05'
+  dpco2: 'spco2_1998-2011_ETH_SOM-FFN_CDIAC_G05'
+
+# cmor standard names
```

Comment:
This should be read from the cmor tables, specifying them here is unnessary and error prone

Reply:
no, these are raw obs files and they can have a myriad of names and naming conventions

Reply:
If there is no associated cmor table, the result from cmorization cannot be used in esmvaltool anyway

Reply:
@bouweandela Are we supposed to create cmor like table of custom data or cmorize custom data?

Reply:
the problem here is two-ways and I see where @bouweandela is coming from: if the short_name of the variable is indeed in the CMOR tables (whether it be standard tables or custom tables) then the standard_ and long_ names should indeed be retrieved from the CMOR tables rather than from this cfg file; but the flip-side is if the short_name is for a completely new variable - does the user run the cmorization first and then adds the variable to the cmor custom tables or the other way around?There is also the aspect of multiple short_names that can be mapped differently depending on the needs of the user

Reply:
@bouweandela as pointed out by @valeriupredoi these are needed to assign the correct input RAW file for each variable and, beside that, there are no other links to CMOR tables

https://github.com/ESMValGroup/ESMValTool/blob/9aafeacb8e64a8f837de9d979e68847c76379197/esmvaltool/utils/cmorizers/obs/cmor_config/Landschutzer2016.yml#L25-L29

In such a way the cmorizer is enough flexible to be adapted for the various types of RAW input data.